### ISSUE TITLE
Usability assessment

### ISSUE THREAD

**Issue by Dahianna** [GSA/digitalcorps.gsa.gov#318]
**Fellows**: Feel free to start working on these issues by assigning them to yourself. If you have any questions feel free to add it in the comments and tag @jaredbain and @Dahianna. Also always create a new branch and add Jared as reviewer.
**Skills:** Data and analytics, project management, content design

As we continue to upgrade the site we want to ensure new and old content is still meeting 508 and other site usability standards. We are currently using AMP.

- Check with Jared on any new issues brought up by the most recent AMP test #269 
- Review  and fix Google Search console [mobile usability report and issues](https://search.google.com/search-console/mobile-usability?resource_id=https://digitalcorps.gsa.gov/&utm_source=wnc_10030322&utm_medium=gamma&utm_campaign=wnc_10030322&utm_content=msg_100058679&hl=en)
- Run any other usability tests that we could be missing and provide a report.

**Comment by EvaDrans** [comment]
#327 Links do not have text.

**Comment by Dahianna** [comment]
We're currently undergoing a usertesting.gov accessibility assessment. More here: #510
